doc/visualizations.rst

```diff
@@ -8,37 +8,83 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and
+input data (`X`, `y`).
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`), which is useful when you only want to compute the
+predictions once.
+
+Using `from_predictions` avoids having to recompute predictions, but does not
+automatically resolve some ambiguities. For binary classification, the user must know
+which column corresponds to the positive label (in this case, `y_pred`).
```

Comment:
Not sure what your thoughts are on this @glemaitre  but for decision_function, I thought we only returned one column? And we need to multiply by -1 if we wanted to switch pos_label?
I don't love the phrase "resolve some ambiguities", I would prefer something direct like, "must pass prediction values corresponding to the `pos_label`, for binary classification. For probabilities this means....For decision function values this means..."

Reply:
@lucyleeow do you agree with the last sentence from Guillaume?

Reply:
Hmm tricky, because the positive label is not always `1` right? E.g., data input to an estimator, the binary labels are not always 0,1, they could be any other 2 values right?

That's why I used the term 'switch labels' though I can see that this is not 100% clear either...